### ISSUE TITLE
Docs that render locally don't format correctly on site

### ISSUE THREAD

**Issue by ttrahan** [microservices-demo/microservices-demo#431]
For docs page:
https://github.com/microservices-demo/microservices-demo/blob/master/docs/deployment/ecs-weave-shippable.md

On site - https://microservices-demo.github.io/microservices-demo/deployment/ecs-weave-shippable.html: 
![image](https://cloud.githubusercontent.com/assets/4467650/20573981/4059031c-b167-11e6-866f-8f1300759424.png)

Rendered locally - http://0.0.0.0:4000/microservices-demo/deployment/ecs-weave-shippable.html:
![image](https://cloud.githubusercontent.com/assets/4467650/20573996/530b4060-b167-11e6-9ef6-6264d18ca5e2.png)

Here is the relevant markdown for this example:
```
### Goal
In this scenario, you'll set up and run the containerized Socks Shop
eCommerce application, with fully automated deployments, in 30
minutes or less.

Your sample application will feature:
  * **Amazon ECS**{: style="color: orange"} for container orchestration
  * **Amazon ECR**{: style="color: orange"} for container registry
  * **Weave Scope**{: style="color: orange"} for service discovery and container visualization
  * **Shippable**{: style="color: orange"} for automated CI/CD

This page provides the instructions necessary to:
  * Provision a Weave-enabled Amazon ECS cluster using Cloud Formations and
  deploy all components of the Sock Shop microservices demo application except
  the <a href="https://github.com/microservices-demo/front-end" style="color: orange">front-end</a>
   component
  * Configure a CI/CD pipeline to manage deployment of the
  <a href="https://github.com/microservices-demo/front-end" style="color: orange">
  front-end</a> service using Shippable
```

**Comment by ttrahan** [comment]
Another example:

On site:
![image](https://cloud.githubusercontent.com/assets/4467650/20574179/245c9628-b168-11e6-8005-6cfdd0458053.png)

Rendered locally:
![image](https://cloud.githubusercontent.com/assets/4467650/20574210/3f7982fe-b168-11e6-93d0-a7f06064cedf.png)

markdown for this example:
```
    Now, load your Pipeline configuration files into Shippable:
    * Select the `Pipelines` tab, `Resources` view, and then `Add Resource`
    button (far right)
    * In the Subscription Integrations dropdown: choose `Add integration` and
    complete the fields, as follows:
      * Name: name your integration `github`
      * Account Integrations: select `github` from the list
      * Projects Permissions: leave as `All projects`
      * Select `Save`
    * Select Project dropdown: choose `microservices-demo` project
    * Select Branch dropdown: choose `master`
    * Select `Save`
    * Select `SPOG` view and verify that your pipeline has been loaded
```

**Comment by ttrahan** [comment]
GitHub Pages uses Kramdown 1.11.1, while local-server.sh installs 1.12.

**Comment by ttrahan** [comment]
I worked around the issue by adding a blank line between the text and first bullet to get it to render properly in kramdown 1.11.1.

Note, to render the site locally with dependencies that match GitHub Pages, including kramdown 1.11.1, I used https://github.com/github/pages-gem. This approach isn't ideal as a warning is presented indicating that it isn't fully compatible with GitHub Pages recent upgrade to Jekyll 3.0.
